src/app/pages/update-item/update-item.page.ts

```diff
@@ -42,7 +42,8 @@ export class UpdateItemPage implements OnInit {
       id: this.item.id,
       version: Number(this.item.version), // For some reason it's a string
       title: value.title,
-      description: value.description
+      description: value.description,
+      status: this.item.status
```

Comment:
I do not see status added into `update-item-page`

Reply:
We're not using the status on update item page, but we see a warning if we don't provide it. What would you recommend?